com.Audentia.BlocNotes/com.Audentia.BlocNotes/DetailViewController.m

```diff
@@ -50,6 +51,26 @@ - (IBAction)shareNote:(id)sender {
     
 }
 
+    
+- (IBAction)didTapEditButton:(id)sender {
+
+    switch (self.noteText.editable) {
```

Comment:
Oh I dunno, I guess I was thinking that I wouldn't have to write as much since it's just checking same condition (self.notetext.editable) and initially I didn't want to just say else because I wanted to catch like the fringe case (in case it was nil) but now that I think about it it's a bool so that can't happen right? Haha so it should have just been if else. 

Sent from my iPad

> On Jun 24, 2015, at 9:25 PM, Falko Buttler <notifications@github.com> wrote:
> 
> In com.Audentia.BlocNotes/com.Audentia.BlocNotes/DetailViewController.m:
> 
> > @@ -50,6 +51,26 @@ - (IBAction)shareNote:(id)sender {
> >      
> >  }
> >  
> > +    
> > +- (IBAction)didTapEditButton:(id)sender {
> > +
> > +    switch (self.noteText.editable) {
> Why don't you just use a regular if/else? A switch with YES and NO cases seems a bit unorthodox.
> 
> —
> Reply to this email directly or view it on GitHub.
>